### ISSUE TITLE
Setting Wifi channel, Hostname or SSID through admin interface while on wifi reports an error

### ISSUE THREAD

**Issue by edwinsteele** [ConnectBox/connectbox-pi#156]
Right after updating the channel, the admin interface says: "Error updating channel” - Unexpected error setting property: error”. Same applies for updating the hostname or SSID. The channel/ssid are successfully updated though.

This doesn’t happen when you’re connected via ethernet, but if you’re connected via wifi, the wifi will drop and if the device connects to another network the admin interface will never get the SUCCESS response back from the API

**Comment by edwinsteele** [comment]
Commandline exit status:
```
root@pilot1:~# /usr/local/connectbox/bin/ConnectBoxManage.sh get channel
5
root@pilot1:~# echo $?
0
root@pilot1:~# /usr/local/connectbox/bin/ConnectBoxManage.sh set channel 3
SUCCESS
root@pilot1:~# echo $?
0
root@pilot1:~# /usr/local/connectbox/bin/ConnectBoxManage.sh get ssid
new ssid
root@pilot1:~# /usr/local/connectbox/bin/ConnectBoxManage.sh set ssid pilot1.connectbox
SUCCESS
root@pilot1:~# echo $?
0
```

**Comment by GeoDirk** [comment]
I noticed that even though it reports and error, it actually makes the change regardless.  Do you think this has anything to do with that the actual SSID has been modified and since you are connecting over WiFi, we end up not being connected which throws the error?  This would explain why doing over ethernet works and WiFi doesn't.

**Comment by kldavis4** [comment]
Yes, that is exactly the issue. There may be something we can do in the admin ui to mitigate this.

**Comment by edwinsteele** [comment]
Fixed by #162
